tests/e2e/tests/pilot/testdata/v1alpha2/istio-rbac-rules.yaml.tmpl

```diff
@@ -57,7 +57,7 @@ metadata:
   name: access-headless
 spec:
   rules:
-  - services: ["headless"]
+  - services: ["headless*"]
```

Comment:
Do you know the name of headless service?

Reply:
It should have the same pattern as other services (i.e. headless.namespace.svc.cluster.local), but I intentionally used * here to avoid any mismatch as we're not testing it anyway.